tests/scripts/history_checks.bash

```diff
@@ -0,0 +1,68 @@
+#! /bin/bash
+
+function check_bin_in_commit() {
+
+    binary_files=""
+
+    changes=$(git diff --pretty=format:"" --shortstat HEAD^ \
+              | sed -e 's/[^0-9,]//g' -e 's/,/+/g' | bc)
+
+    if (( $changes > 50000 ))
+    then
+        msg "$(git log -1 --format=%H)"
+        msg "ATTENTION: This commit is unusually large"
+        return 1
+    fi
+
+    return 0
+}
+
+function check_commit_size() {
+
+    binary_files=""
+
+    if [[ -n ${binary_files} ]]
+    then
+        msg "$(git log -1 --format=%H)"
+        msg "ATTENTION: binary file(s) added or modified"
+        echo "${binary_files}"
```

Comment:
We don't just want to know there is a problem, but also what is the source of it. So I will rework this message similar to the above, making sure to list all the names of all the binary files.

Reply:
I think that’s the point of the last line, isn’t it?